packages/block-editor/src/components/child-layout-control/index.js

```diff
@@ -188,6 +192,7 @@ function FlexControls( {
 						} );
 					} }
 					value={ flexSize }
+					min={ minUnitValue }
```

Comment:
```suggestion
					min={ 0 }
```

Suggestion: We can just hardcode value here and there's no need to pass `minUnitValue`. I cannot think of a case where negative height/width values should be allowed here.

Reply:
The thought behind not hardcoding was to provide maximum flexibility. Thanks for the simplification :)

Reply:
There are still some leftovers for the `minUnitValue` prop. We should also remove those, as they serve no purpose now.

Reply:
Yes, @Mamaduka, I am on it.